src/server/bootstrap.ts

```diff
@@ -66,6 +67,18 @@ const cors = require('cors');
 
 Resource.validate = validate;
 
+const options = {
+  concurrency: Number(
+    process.env.PROJECT_FILTERS_THREADS_POOL_CONCURRENCY || 1,
+  ),
+  maxQueuedJobs: Number(
+    process.env.PROJECT_FILTERS_THREADS_POOL_MAX_QUEUED || 4,
```

Comment:
Why do we override its default unlimited value? If we get many queries simultaneously it will cause issue.

Reply:
I was probably thinking unlimited stuff is bad hahaha

but I will remove the limit as we don't want now to break the query when we have high loads of people, we want performance not new issues.